docs/DeveloperGuide.md

```diff
@@ -4,30 +4,148 @@
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
+* [JSON-java](https://github.com/stleary/JSON-java)
+* [three-ten-extra](https://www.threeten.org/threeten-extra/)
+* * Requesting and Parsing of data from API into Java
```

Comment:
Appears to be a duplicated bullet point in this line, possible formatting error?